### PR TITLE
Updated README.md

### LINKED ISSUE #30: Update README.md
Create a proper README.md file that includes all the important details of the project. Also add screenshots.

### PR DESCRIPTION
**Pull request by SubhradeepSS**
Fixes #30 

Changes: Updated the README.md file for the repository of the app.
Link : https://github.com/SubhradeepSS/RoboTalk/blob/development/README.md

### CODE REVIEW

**Comment by SubhradeepSS** [comment]
@somenath1435 please review my PR for the issue #30

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
 # RoboTalk
 An Android app to convert text to speech and vice versa.
+
+## Features
```
Comment (ShivangiSingh17) [inline README.md:4]: Can you add a section of "Getting started" that tells how to set the development environment for the contributors to work on the project.

**Review by ShivangiSingh17 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SubhradeepSS) [reply README.md:4]: @ShivangiSingh17 added the "Getting Started" Section.
please review and suggest any changes applicable.

**Review by ShivangiSingh17 — APPROVED** [review]

**Comment by ShivangiSingh17** [comment]
Good work @SubhradeepSS :)

**Comment by SubhradeepSS** [comment]
> Good work @SubhradeepSS :)
@ShivangiSingh17
Can u please update my score in the leaderboard?

**Pull request merged**